docs/reference/technology/USB.md

```diff
@@ -56,11 +56,11 @@ USB defines 5 separate states a device can be in - Attached, Powered, Default, A
 | Address    |    No new functionality                  |
 | Configured |    All enabled endpoints are functional  |
 
-At any time, the USB device can enter a state with less functionality. This could be due to a loss of power event or a surprise USB disconnect. When leaving or outside of the Configured state, USBDevice ignores writes to and reads from all endpoints other than endpoint 0.
-
+The USB device can enter a state with less functionality at any time. This could be due to a loss of power or surprise disconnect. When leaving or outside of the Configured state, USBDevice ignores writes to and reads from all endpoints other than endpoint 0.
```

Comment:
Looking at it again, I think we're talking more *random* occurrences of power loss or disconnects, rather than 'at any time', which seems to imply it's a specific fault of USB.

Reply:
```suggestion
Random power loss or disconnect can cause the USB device to lose functionality. When leaving or outside of the Configured state, USBDevice ignores writes to and reads from all endpoints other than endpoint 0.
```

Reply:
Does that communicate the idea more clearly?